### DISCUSSION TITLE
Find more opportunities for dissemination and community engagement

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#374] [General]
## Summary

It would be fantastic to find more opportunities to promote the project, e.g. webinars, seminars, conferences.

## Date Due by

Ongoing

## Who can help

Anyone can help by writing or nominating any opportunities to promote and disseminate the project, as well as to suggest communitis to engage with or relevant courses for appropriate learning. Just comment on this issue with suggestions.
Ideal is any opportunities which would be a way of fostering the diversity of the community

**Comment by GeorgiaHCA** [comment]
Interesting article on involving people with learning disabilities in the design process: https://www.tandfonline.com/doi/abs/10.1080/15710882.2014.997829

**Comment by lottycoupat** [comment]
If this research and the data the platform collects aims to help with policy making - have we thought of potentially doing some talks/webinars to the Civil Service? I could try and find some connections through my Fast Stream networks?

**Comment by GeorgiaHCA** [comment]
Great idea @lottycoupat - I think @JamesSCTJ would be a great person to work on this with (if you both have time?). I've also been in touch with some people from GDS who might be able to help adapt some of their user-tested components for the wireframe. 

Given the civil service has a lot of development/volunteering opportunities, corporate objectives, etc., we could potenitally offer an opportunity for development and gain support building the platform.
